src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -198,6 +198,7 @@ internal static class IDEDiagnosticIds
     public const string UseCollectionExpressionForCreateDiagnosticId = "IDE0303";
     public const string UseCollectionExpressionForBuilderDiagnosticId = "IDE0304";
     public const string UseCollectionExpressionForFluentDiagnosticId = "IDE0305";
+    public const string UseInterpolatedStringDiagnosticId = "IDE0330";
 
     public const string MakeAnonymousFunctionStaticDiagnosticId = "IDE0320";
```

Comment:
```suggestion

    public const string MakeAnonymousFunctionStaticDiagnosticId = "IDE0320";
    
    public const string UseInterpolatedStringDiagnosticId = "IDE0330";
```

Reply:
Fixed in https://github.com/dotnet/roslyn/pull/73008/commits/b586235e2b496d486d5888c46b60b7fe245a1275